recipes/openblas/all/conanfile.py

```diff
@@ -0,0 +1,84 @@
+from conans import ConanFile, CMake, tools
+import sys
+import os
+
+
+class OpenBLAS(ConanFile):
+    name = "openblas"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.openblas.net"
+    description = "An optimized BLAS library based on GotoBLAS2 1.13 BSD version"
+    topics = (
+        "openblas",
+        "blas",
+        "lapack"
+    )
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "build_lapack": [True, False]
```

Comment:
I think we should have this as a recipe in conan-center and avoid this one building a library that has entity on its own. WDYT @uilianries?

Reply:
Totally @danimtb ! We already have a lapack recipe in Conan Community, which can be imported to CCI: https://github.com/conan-community/conan-lapack

Reply:
Aren't they two different lapack libraries though?

Reply:
this topic is a bit complicated. there are APIs, like BLAS, CBLAS, LAPACK, LAPACKE, etc.
there is reference (so called **netlib**) LAPACK implementation, which [conan-lapack](https://github.com/conan-community/conan-lapack) package provides.
on other hand, OpenBLAS library implements many of previously mentioned APIs on its own (e.g. BLAS, CBLAS). besides that, as far as I understand, OpenBLAS contains original reference (netlib) lapack code with some functions optimized.
see comments:
https://github.com/xianyi/OpenBLAS/issues/203
```
By default, it includes a full LAPACK lib (with some parts optimized).
```
https://github.com/xianyi/OpenBLAS/issues/182
```
OpenBLAS implements optimized variants of a subset of the LAPACK functions
```
OpenBLAS and netlib lapack are not only libraries providing LAPACK functions, e.g. Intel MKL has such implementations. 
the major downside of the package is requirement of fortran compiler to use LAPACK API. I am not sure how to deal with that yet.

Reply:
> the major downside of the package is requirement of fortran compiler to use LAPACK API. I am not sure how to deal with that yet.

@SSE4 If you are talking about this package, is the warning about requiring a fortran compiler and the eventual compilation fail not enough for now? Or would you like me to remove the option for the time being?

Reply:
@danimtb do we have fortran available in CCI? what's our policy about it?

Reply:
Any updates?

Reply:
Sorry for the late reply. We don't have the Fortran compiler at this moment but we should consider adding it regarding packages like this one and LAPACK. We will have to discuss it

Reply:
@danimtb thanks for the update :).
Is it unreasonable to add the recipe in with the lapack build turned off by default for now? Not all use cases of openblas require lapack support (mine doesn't), and users can still build it themselves should they require it.

Reply:
That's fine, we can move this forward and print an error for that option (or just remove it) if openblas does not require fortran (AFAIK it doesn't)

Reply:
let's move forward, will discuss fortran later at the time we add LAPACK